cloudflare/resource_cloudflare_zone_settings_override.go

```diff
@@ -415,9 +415,17 @@ var resourceCloudflareZoneSettingsSchema = map[string]*schema.Schema{
 	},
 
 	"http2": {
-		Type: schema.TypeString, ValidateFunc: validation.StringInSlice([]string{"on", "off"}, false),
-		Optional: true,
-		Computed: true,
+		Type:         schema.TypeString,
+		ValidateFunc: validation.StringInSlice([]string{"on", "off"}, false),
```

Comment:
This puts `ValidateFunc` on it's own line instead of inline. Best review this with `?w=1` for actual change.